packages/datagrid/src/components/DataGrid/DataGrid.component.js

```diff
@@ -2,9 +2,7 @@
 import React from 'react';
 import classNames from 'classnames';
 import keycode from 'keycode';
-import { AgGridReact } from 'ag-grid-react';
-import 'ag-grid-community/dist/styles/ag-grid.css';
-
+import assetsApi from '@talend/assets-api';
```

Comment:
#4002 opened 20 days ago mentionned that this was not stable (last commit on this lib dates 24 days ago), is it safe to use it in the end?

Reply:
the only bug found AFAIK is fixed here.
until we find other bugs like #3970 for example which impact only version bump not already release.

If you are aware of other issues, do not hesitate to report them, will do my best to fix them.